src/WebJobs.Script/Environment/EnvironmentExtensions.cs

```diff
@@ -115,6 +115,12 @@ public static bool AzureFilesAppSettingsExist(this IEnvironment environment)
                    !string.IsNullOrEmpty(environment.GetEnvironmentVariable(AzureFilesContentShare));
         }
 
+        public static bool IsDotNetInProcSupported(this IEnvironment environment)
+        {
+            return !IsFlexConsumptionSku(environment)
+                   && !string.Equals(environment.GetEnvironmentVariable(EnvironmentSettingNames.FunctionsDisableInProc), "1", StringComparison.OrdinalIgnoreCase);
+        }
```

Comment:
Does `EnvironmentSettingNames.FunctionsDisableInProc` default to true for FlexConsumption or does it have to be set to true in dockerfile for FlexConsumption image?

Reply:
No. This gives you the ability to set this outside of Flex or an environment where a hosting config is applied.